PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -145,6 +145,16 @@
     modifier.toModify(extraFlagsTable, variables= cms.PSet())
     modifier.toModify(extraFlagsTable, variables = dict(Flag_ecalBadCalibFilterV2 = ExtVar(cms.InputTag("ecalBadCalibFilterNanoTagger"), bool, doc = "Bad ECAL calib flag (updated xtal list)")))
 
+import RecoTauTag.RecoTau.tools.runTauIdMVA as tauIdConfig
```

Comment:
Add comment about goal of the new function, something like "Tool to add new TauIDs" or so. 
I propose to keep previously used name of updated tau collection, i.e. `slimmedTausUpdated` (check also other places).